packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.tsx

```diff
@@ -102,6 +108,10 @@ export class GuxTimePicker {
 
     this.i18n = await buildI18nForComponent(this.root, translationResources);
     this.clockType = this.clockType || getLocaleClockType(this.root);
+
+    if (this.clockType == '12h' && (this.min || this.max)) {
+      console.error('clock type must be "24h" when using min/max props');
```

Comment:
@katie-bobbe-genesys I had this code here already to check if the clock type is 12h but I used a console error. Should I keep this code but change to a console warn? I'll also do an `if` check like you mentioned to ignore the min/max logic in a certain scenario.